src/components/MovieCard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <article class="movie-card" v-on:click="clickMovie">
-     <img v-if="!movie.poster.includes('null')" :src="movie.poster" alt="" class="poster"/>
+  <article class="movie-card" >
+     <img v-if="!movie.poster.includes('null')" :src="movie.poster" alt="" class="poster" v-on:click="clickMovie"/>
```

Comment:
Good switch. The modal was popping up anywhere i clicked even if it was on other buttons.